skyVegasTests/soundCheckGameClient.spec.ts

```diff
@@ -17,22 +23,23 @@ readGames().forEach((game) => {
     });
 
     test("Test sound toggle", async () => {
-      const soundToggleGameWindow = page.frameLocator("#root iframe").locator("i").nth(2)
+      const soundToggleIcon = page
+        .frameLocator("#root iframe")
+        .locator("i")
+        .nth(2);
       startEventListener(page, consoleMessages);
       await launchGame(page, game, consoleMessages);
-      if (messageExists(consoleMessages, ExpectedMessage.SOUND_CHECK_TOOL_BAR_ON)) {
-        await soundToggleGameWindow.click();
-        await validateConsoleMessages(
-          ExpectedMessage.SOUND_CHECK_TOOL_BAR_OFF,
-          consoleMessages
-        );
-      } else if (messageExists(consoleMessages, ExpectedMessage.SOUND_CHECK_TOOL_BAR_OFF)) {
-        await soundToggleGameWindow.click();
-        await validateConsoleMessages(
-          ExpectedMessage.SOUND_CHECK_TOOL_BAR_ON,
-          consoleMessages
-        );
-      }
+      await soundToggleIcon.click();
+      await soundToggleIcon.click();
+      await validateConsoleMessages(
+        ExpectedMessage.SOUND_CHECK_GAME_ON,
+        consoleMessages
+      );
+      await soundToggleIcon.click();
```

Comment:
Maybe we should check that sound game is OFF first and then click again and check sound game ON. That we we call soundToggleIcon.click() twice only instead of 3 times?

Reply:
This was a typo, removed it.